packages/js/components/src/advanced-filters/stories/index.js

```diff
@@ -4,7 +4,7 @@
 import { AdvancedFilters } from '@woocommerce/components';
 
 const ORDER_STATUSES = {
-	cancelled: 'Cancelled',
+	cancelled: 'Canceled',
```

Comment:
I think it's just for the storybook. Do we need to change this? (Also other ./stories changes)